cg/cli/upload/nipt/statina.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from pathlib import Path
+
+import click
+
+from cg.meta.upload.nipt.nipt import NiptUploadAPI
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def statina():
+    """Upload NIPT result files"""
+    pass
+
+
+@statina.command("case")
+@click.argument("case_id", required=True)
+@click.option("--dry-run", is_flag=True)
+@click.pass_obj
+def batch(configs: CGConfig, case_id: str, dry_run: bool):
+    """Loading batch into the NIPT database"""
+
+    LOG.info("*** Statina UPLOAD START ***")
+
+    nipt_upload_api = NiptUploadAPI(configs)
+
+    try:
+        hk_results_file: str = nipt_upload_api.get_housekeeper_results_file(
+            case_id=case_id, tags=["nipt", "metrics"]
+        )
+        results_file: Path = nipt_upload_api.get_results_file_path(hk_results_file)
+
+        hk_multiqc_file: str = nipt_upload_api.get_housekeeper_results_file(
+            case_id=case_id, tags=["nipt", "multiqc-html"]
+        )
+        multiqc_file: Path = nipt_upload_api.get_results_file_path(hk_multiqc_file)
+
+        hk_segmental_calls_file: str = nipt_upload_api.get_housekeeper_results_file(
+            case_id=case_id, tags=["nipt", "wisecondor"]
+        )
+        segmental_calls_file: Path = nipt_upload_api.get_results_file_path(hk_segmental_calls_file)
```

Comment:
I suggest moving this into the upload_to_statina method so all the file fetching happens in the API